resources/assets/app.js

```diff
@@ -5,11 +5,18 @@ import React from 'react';
 import ReactDom from 'react-dom';
 
 import CampaignOverview from './components/CampaignOverview'
+import CampaignInbox from './components/CampaignInbox'
 
 ready(() => {
   const overviewContainer = document.getElementById('overviewContainer');
+  const inboxContainer = document.getElementById('inboxContainer');
+
 
   if (overviewContainer) {
     ReactDom.render(<CampaignOverview {...window.STATE} />, overviewContainer);
   }
+
+  if (inboxContainer) {
```

Comment:
I'm worried about this getting crazy, always having to check if the container is on the page before injecting the js. @DFurnes  [had an idea](https://github.com/DoSomething/rogue/pull/152#issuecomment-284784711) to get around this. Not sure if this is critical at the moment but I worry this will become a pattern.